duckduckgo/iOS
# 18. Subscription Entitlements caching

Task/Issue URL: https://app.asana.com/0/1204099484721401/1206574474409573/f

**Description**:
- This PR just bumps BSK (No changes required on iOS)
- MainActor change has no relation to subscriptions - iOS app was broken in `main` because of a change in BSK, this fixes that



**Steps to test this PR**:
1. 1. Check instructions in [BSK PR](https://github.com/duckduckgo/BrowserServicesKit/pull/703)
